Stdin/README.md

```diff
@@ -0,0 +1,123 @@
+# Stdin Interface
+
+**Status**: Proposed
+
+| Name      | ID                                                                |
+| :-------- | :---------------------------------------------------------------- |
+| Stdin     | 8796536366ee21852db56dccb60bc564598b618c865fc50c8b1ab740bba128e3  |
+
+
+## Description
+
+Allows to get input from user for basic types like integers, addresses, public keys, strings, cells.
+
+## Functions
+
+`getStr` - prints prompt message to the user and returns string entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: bytes - string entered by user
+
+`getInt` - prints prompt message to the user and returns singed integer entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: int256 - integer entered by user
+
+`getUint` - prints prompt message to the user and returns unsinged integer entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: uint256 - integer entered by user
+
+`getAddr` - prints prompt message to the user and returns address entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: uint256 - signed integer entered by user
+
+`getCell` - prints prompt message to the user and returns TON address entered by user
```

Comment:
I see a copy/paste problem (getCell description is copied from getAddr).
And we need an examples of the use cases for each get.

Reply:
Maybe, separate `getContractCode` / `getContractStateInit` with requesting tvc-file from user.

Reply:
@azhogin It should be a separate interface to work with file system and load tvc files. `getCell` - a very simple input, it will be used for special purposes like debugging maybe.